kernel/fs/exfat.inc

```diff
@@ -17,7 +17,7 @@ iglobal
 align 4
 exFAT_user_functions:
         dd      exFAT_free
-        dd      (exFAT_user_functions_end - exFAT_user_functions - 4) / 4
+        dd      (exFAT_user_functions_end - exFAT_user_functions - 8) / 4
```

Comment:
Can you please explain where this number 8 comes from?

Reply:
8 - offset from the first export function for the user api.

struct fs_user_func
fs_free
count_user_func
0_user_func
...
n_user_func
ends

 \+ 4 = skip fs_free
 \+ 8 = skip fs_free and count_user_func.
An error was detected during a test rename on an NTFS partition

Reply:
> 8 — смещение от первой функции экспорта для пользовательского API.
> 
> struct fs_user_func fs_free count_user_func 0_user_func ... n_user_func заканчивается
> 
> + 4 = пропустить fs_free + 8 = пропустить fs_free и count_user_func. Обнаружена ошибка во время тестового переименования раздела NTFS.

Can we replace it with a constant?

Reply:
for example

```
exFAT_user_functions:
        dd      exFAT_free
        dd      (.end - .first) / 4
.first:
        dd      exFAT_0_func
        dd      exFAT_1_func
        dd      exFAT_2_func
.end:
```

Reply:
@Doczom Can you fix it this way?

Reply:
> @Doczom Can you fix it this way?

in new commit or fixed old commit ?

Reply:
Preferably of course, stick to the previous one. It's simple. 
You need to make new changes on your branch, then `git add` and `git commit --amend --no-edit`. And make a force push.

Reply:
You'll have to rebase and fix it in this commit [Krn: Fixed the number of functions in the file system drivers](https://github.com/kolibrios-nextgen/kolibrios-nextgen/pull/37/commits/fe28e55660546a4a1ae573f2353707dfe759da55)

Reply:
> You'll have to rebase and fix it in this commit [Krn: Fixed the number of functions in the file system drivers](https://github.com/kolibrios-nextgen/kolibrios-nextgen/pull/37/commits/fe28e55660546a4a1ae573f2353707dfe759da55)

done

Reply:
@Doczom Sorry about this, but I would like to ask you to fill out the “Testing” section in the PR description. I really don't understand how to check this. I also ask you to add an extended description of why this was done. Literally, "what didn’t work", but "what works now". Thank you in advance